1-starter/README.md

```diff
@@ -88,8 +92,8 @@ For the full command list please view the [Docker Documentation](https://docs.do
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in [Discord Server](https://discord-ts.js.org/discord)
+- Ask in the community [Discord server](https://discord-ts.js.org/discord)
 
 # 💖 Thank you
 
-Show your support for [Discordx](https://www.npmjs.com/package/discordx) by giving us a star on [GitHub](https://github.com/oceanroleplay/discord.ts).
+Why not show your support for [Discordx](https://www.npmjs.com/package/discordx) by giving us a star on [GitHub](https://github.com/oceanroleplay/discord.ts).
```

Comment:
revert

Reply:
Old message sounds more like a command than something the reader could do.

Reply:
`You can support Discordx by giving it a GitHub star`?

Reply:
Sounds better. I'll use that instead, it keeps the tone of the original while not sounding like a command xd

Reply:
same would be needed on `discord.ts` repo as well

Reply:
for `discord.ts` you will fine a simillar line in ``create-discordx`` source code, we should cover it as well